02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/02 Tensorflow/06 Predict Labels.html

```diff
@@ -0,0 +1,14 @@
+<p>To predict the labels of new data, in the <code>OnData</code> method, get the most recent set of features and then call the <code>run</code> method with new features.</p>
+<div class="section-example-container">
+    <pre class="python">new_features, __ = self.get_features_and_labels()
+prediction = self.model.run(self.output, feed_dict={self.X: new_features})
```

Comment:
Instead of making predictions on all the feature samples, just use the last row (the most recent 5 closing prices)